doc/examples/filters/plot_cycle_spinning.py

```diff
@@ -0,0 +1,89 @@
+"""
```

Comment:
@hmaarrfk did this file, cycle spinning, and hysteresis thresholding get caught in here by mistake? I don't think cycle spinning at least is in the 0.13 branch, though maybe I'm wrong...

Reply:
Your right. I'm glad I'm not trying to backport the other big patch I submitted ;).